src/main/java/org/kitteh/irc/client/library/feature/twitch/messagetag/MsgParamCumulativeMonths.java

```diff
@@ -0,0 +1,35 @@
+package org.kitteh.irc.client.library.feature.twitch.messagetag;
```

Comment:
Missing license header.

Reply:
This can be resolved by just running maven once, FYI :)